kombu/asynchronous/semaphore.py

```diff
@@ -28,13 +40,18 @@ class LaxBoundedSemaphore:
         HELLO 3
     """
 
-    def __init__(self, value):
+    def __init__(self, value: int) -> None:
         self.initial_value = self.value = value
-        self._waiting = deque()
+        self._waiting: Deque[tuple] = deque()
```

Comment:
Maybe? 🤔
```suggestion
        self._waiting: Deque[Tuple[callable, list, dict] = deque()
```

Reply:
This unfortunately causes issues with the ParamSpec annotations further down:

```
kombu/asynchronous/semaphore.py:67: error: Argument 1 has incompatible type "Tuple[Callable[P, None], P.args, P.kwargs]"; expected "Tuple[Callable[..., Any], List[Any], Dict[Any, Any]]"  [arg-type]
Found 1 error in 1 file (checked 1 source file)
```